lib/options-manager.js

```diff
@@ -309,6 +309,10 @@ const buildESLintConfig = options => config => {
 		};
 	}
 
+	if (options.fixDryRun) {
```

Comment:
I think this could use a code comment on how it works. It's not immediate obvious how it works since you only see here that it sets `.fix = true`.